cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/containers/NotifyContextRequest.java

```diff
@@ -352,7 +352,11 @@ public String getContextValue() {
                 return  "" + value;
             }  // if then else if
         } // getContextValue
-        
+
+        public JsonElement getValue() {
```

Comment:
Although this functions is pretty simple, I think style is to include Javadoc (and "closing } comment") in every function (not fully sure, should be checked...).

Reply:
Fixed in [93df3b5](https://github.com/telefonicaid/fiware-cygnus/pull/1784/commits/93df3b537079ac6994c49c6bce1f432c5aa3b1ff)